catalog/core/catalog-core-impl/pubsub/src/main/java/ddf/catalog/pubsub/EventProcessorImpl.java

```diff
@@ -119,7 +119,7 @@ public static void processEntry(Metacard metacard, String operation, EventAdmin
             LOGGER.debug("catalog ID = {}", metacard.getId());
             LOGGER.debug("operation = {}", operation);
 
-            HashMap<String, Object> properties = new HashMap<String, Object>();
+            HashMap<String, Object> properties = new HashMap<String, Object>(3, 1);
```

Comment:
Size and load are a little unusual to specify, especially for such small maps. Was there a specific reason for this?